src/DeploymentUtils.ts

```diff
@@ -6,7 +6,11 @@ interface DeploymentExport {
 const deployments: DeploymentExport = deployments_ as any;
 
 // Returns the deployed address of any contract on any network.
-export function getDeployedAddress(contractName: string, networkId: number, throwOnError = true): string | undefined {
+export function getDeployedAddress(
+  contractName: string,
+  networkId: number | string,
```

Comment:
Network id `1810017368444177321` (solana-devnet) overflows js numbers so we need to pass it as a string